src/buildHTML.js

```diff
@@ -1491,6 +1491,27 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.not = function(group, options, prev) {
+    const inner = buildExpression(group.value.body, options);
+    const solidus = buildCommon.mathsym(
+        "\\not", group.mode, options.getColor(), ["mrel"]);
+
+    // 0.778 = -1 * ptPerEm * -778
+    // -778 is the horizontal correction that has been added to glyph.
+    // TODO(kevinb) update the font to remove this correction so that we don't
+    // have to reverse its effects here.
+    solidus.style.position = 'absolute';
+    const transform = 'translate(0.778em, 0em)';
+    solidus.style.transform = transform;
+    solidus.style.MozTransform = transform;
+    solidus.style.MsTransform = transform;
+    solidus.style.WebkitTransform = transform;
```

Comment:
Why use a transform instead of `solidus.style.left = '0.778em'`? Shouldn't that do the same thing?